reflekt-plugin/src/test/resources/org/jetbrains/reflekt/plugin/compiler/additional-sources/common-files/ExpectedReflektClasses.kt

```diff
@@ -0,0 +1,70 @@
+@file:OptIn(InternalReflektApi::class)
+
+package org.jetbrains.reflekt.test.common
```

Comment:
Please, move this file into the helpers folder, because it is not a part of common-project files

Reply:
🆗